{{ cookiecutter.formula_name }}-formula/{{cookiecutter.formula_name}}/service.sls

```diff
@@ -0,0 +1,5 @@
+{{ cookiecutter.formula_name }}_service_running:
```

Comment:
No, the cookiecutter variable is present when generating the initial project structure. For the service state itself we don't want any imports because it gets imported into other states (e.g. install, configure, etc.).